src/main/java/calculator/StringClassifier.java

```diff
@@ -0,0 +1,17 @@
+package calculator;
+
+import java.util.List;
+
+public class StringClassifier {
+    private static final String DELIMITER = " ";
+    public static List<String> split(String text) {
```

Comment:
개행을 추가하면 상수와 메서드를 구분하기 쉽도록 가독성을 높여볼 수 있을 것 같습니다.
```suggestion
    private static final String DELIMITER = " ";

    public static List<String> split(String text) {
```